app/src/main/java/com/codepath/apps/critterfinder/models/PetModel.java

```diff
@@ -69,6 +69,15 @@ public String getSize(){
         return size;
     }
 
+    public String getSizeFullName(){
+        switch (size){
+            case "S": return "Small";
+            case "M": return "Medium";
+            case "L": return "large";
+        }
+        return "";
+    }
+
```

Comment:
I shared some pointer with @scottrichards in another PR about how this can be done in a friendly way for translations by leveraging string-array xml files which is how our Search Filter UI handles this for Small, Medium and Large.

But for a demo this works.